medusa/medusacli.py

```diff
@@ -70,6 +75,7 @@ def configure_logging(verbosity, without_log_timestamp):
 
 @click.group()
 @click.option('-v', '--verbosity', help='Verbosity', default=0, count=True)
+@click.option('--log-file', help='Log to file as well')
```

Comment:
Being able to enforce the logging it at runtime using an argument is a good thing to have, but we could maybe push this a little further by making it part of the configuration. What do you think?
We could have a `log_file=/path/to/logs` in the `medusa.ini` file that would allow to set logging to a file by default.

If you don't feel like doing this in this PR, it's fine, we can add it later as part of another PR.

Reply:
I have added support to `logging` section in the configuration and removed the CLI argument to avoid ambiguity.